utils/calc-stack-height/src/main.rs

```diff
@@ -35,8 +36,25 @@ fn main() -> anyhow::Result<()> {
     let memory = Memory::new(&store, MemoryType::new(0, None, false))?;
     env.insert("memory", Extern::Memory(memory));
 
-    let func = Function::new_native(&store, || {});
-    env.insert("gr_out_of_gas", func);
+    // taken from `gear_sandbox_host::sandbox::wasmer_backend::dispatch_function_v2`
+    #[derive(Default, Clone)]
+    struct Env;
+
+    impl WasmerEnv for Env {
+        fn init_with_instance(&mut self, _: &Instance) -> Result<(), HostEnvInitError> {
+            Ok(())
+        }
+    }
+
+    let ty = FunctionType::new(vec![Type::I32], vec![]);
+    let func = Function::new_with_env(&store, &ty, Env, |_, args| match SystemBreakCode::try_from(
```

Comment:
```suggestion
    let ty = FunctionType::new(vec![Type::I32], vec![]);
    let func = Function::new(&store, &ty, |_, args| match SystemBreakCode::try_from(
```

Reply:
discussed in pm, done in c9099670212f0638bb7353add29b5675a33a3b94